operator-examples/myrealm.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakRealm
+metadata:
+  name: myrealm
+  labels:
+    realm: myrealm
+spec:
+  realm:
+    id: "myrealm"
```

Comment:
Why are you setting "id" here? This is really an internal ID and shouldn't be set here.

Reply:
Removed.